docs/tutorial/get-started.rst

```diff
@@ -70,8 +69,7 @@ The response should look somewhat as shown below:
      Services: mds, mgr, mon
         Disks: 0
 
-Your cluster deployment summary contains your node's hostname (IP address), such as ``ubuntu`` (``10.246.114.49``), along with information about the
-services running and available storage. You'll notice that the cluster is healthy with one node and three services running, but no storage has been allocated yet. 
+Your cluster deployment summary contains your node's hostname (IP address), such as ``ubuntu`` (``10.246.114.49``), along with information about the services running and available storage. You'll notice that the cluster is healthy with one node and three services running, but no storage has been allocated yet. 
```

Comment:
"such as" sounds a bit off. Maybe we should reword this to show that in our case, it is ``ubuntu`` (``10.246.114.49``).